.github/workflows/docker-publish.yml

```diff
@@ -93,6 +97,14 @@ jobs:
       - name: Build Image
         run: |
           python3 build.py -p cass-operator -m time-travel -d $IMAGE_NAMESPACE
+      - name: Sieve Learn
+        run: |
+          python3 sieve.py -p cass-operator -t recreate -m learn -d $IMAGE_NAMESPACE
```

Comment:
Can you do `-s learn` here?
The most recent version only accepts `-s learn` if you want to run the learning stage